test/unit/blocks_looks.js

```diff
@@ -23,6 +25,146 @@ const fakeRuntime = {
 };
 const blocks = new Looks(fakeRuntime);
 
+test('switch costume block runs correctly', t => {
+    /**
+     * Test which costume index the `switch costume`
+     * block will jump to given an argument and array
+     * of costume names.
+     *
+     * @param {string[]} costumes List of costume names as strings
+     * @param {string|number|boolean} arg The argument to provide to the block.
+     * @param {number} [currentCostume=1] The 1-indexed default costume for the sprite to start at.
+     * @return {number} The 1-indexed costume index on which the sprite lands.
+     */
+    const testCostume = (costumes, arg, currentCostume = 1) => {
```

Comment:
`testCostume` and `testBackdrop` can be helpers outside of this test definition. You can also add one more optional argument at the end of testCostume to swap the relevant costume/backdrop bits so that the two functions can share code.

E.g. 
```
const testCostume = (costumes, arg, currentCostume = 1, costumeOrBackdrop = 'costume') => { ... };

const testBackdrop = (backdrop, arg, currentBackdrop = 1) => testCostume(backdrop, arg, currentBackdrop, 'backdrop');
```

I also think this code is okay as is though.